src/algorithms/crypto/randomness.md

```diff
@@ -86,6 +66,36 @@ GetRandomness(dst, l, s):
 {{< readfile file="/docs/actors/actors/crypto/randomness.go" code="true" lang="go" >}}
 {{< readfile file="/docs/systems/filecoin_blockchain/struct/chain/chain.go" code="true" lang="go" >}}
 
+## Drawing tickets from the VRF-chain for proof inclusion
+
+There are a few instances in which the protocol requires inclusion  to be drawn from the Filecoin blockchain's VRF-chain (which generates {{<sref tickets>}} with each new block), in order to tie certain proofs to a particular set of Filecoin blocks (i.e. a given chain or fork). This is notably the case for Seal Pre-commits (see {{<sref sealing>}}).
```

Comment:
Can we list out the instances where it is the case ? It's not obvous right now (to me and I believe @sa8 )

Reply:
Tl;dr sealrandomness is VRF chain all the rest is drand. I believe this can come in a separate PR.

Reply:
Per @nicola's point, I feel like current phrasing should suffice. Otherwise I can do this, but feels a bit restrictive:

```suggestion
There are a few instances in which the protocol requires inclusion  to be drawn from the Filecoin blockchain's VRF-chain (which generates {{<sref tickets>}} with each new block), in order to tie certain proofs to a particular set of Filecoin blocks (i.e. a given chain or fork). This is currently only the case for Seal Pre-commits (see {{<sref sealing>}}).
```